pkgs/applications/misc/tdrop/default.nix

```
@@ -1,31 +1,41 @@
 { stdenv, lib, fetchFromGitHub, makeWrapper
-, xwininfo, xdotool, xprop }:
+, xwininfo, xdotool, xprop, gawk, coreutils
+, gnugrep, procps-ng }:
```

```suggestion
, gnugrep, procps }:
```

`procps-ng` is an alias for `procps` and aliases are forbidden.